layouts/index.html

```diff
@@ -5,46 +5,74 @@
         {{ partial "skip.html" . }}
         {{ partial "nav.html" . }}
         <main id="content">
+            <div class="container p-0">
+                <div id="hp-hero" class="jumbotron-fluid">
+                    <h1 class="display-4">Help your patients who are at risk for opioid addiction.</h1>
```

Comment:
The text overlaying the image + button is hard to read and a somewhat dodgy shade of brown.